babel.config.js

```diff
@@ -8,15 +8,25 @@ module.exports = function (api) {
       browsers: ['chrome >= 89', 'firefox >= 89'],
     },
     plugins: [
-      // `browserify` is old and busted, and doesn't support `??=` (and other
-      // logical assignment operators). This plugin lets us target es2020-level
-      // browsers (except we do still end up with transpiled logical assignment
-      // operators 😭)
+      // This plugin enables support for logical assignment operators in browsers
+      // that may not support them natively.
```

Comment:
This plugin wasn't added for browsers; it is here only for browserify. If browserify wasn't old and busted this wouldn't be needed since our target browsers all support logical assignment operators.